libPhoneNumber/NBGeocoderMetadataHelper.h

```diff
@@ -0,0 +1,27 @@
+//
+//  NBGeocoderMetadataHelper.h
+//  libPhoneNumberiOS
+//
+//  Created by Rastaar Haghi on 6/12/20.
+//  Copyright © 2020 ohtalk.me. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <sqlite3.h>
+#import "NBPhoneNumber.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NBGeocoderMetadataHelper : NSObject
+
+@property(nonatomic, copy) NSString *regionDescription;
+@property(nonatomic, copy) NSNumber *countryCode;
+@property(nonatomic, copy) NSString *language;
```

Comment:
Let's make these readonly. We can make the user create one instance of the metadata helper per language

Reply:
regionDescription has been removed (as it was unnecessary), language has been made readonly, but countryCode cannot be made readonly because it constantly gets updated whenever the searched country changes (and needs the variable to be updated to keep track of when to update the SQL statements).